sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -775,7 +844,7 @@ export interface DecryptResult {
   /**
    * Result of the operation
    */
-  result: Uint8Array,
+  result: Uint8Array;
 }
 
```

Comment:
These interfaces are missing the other details we have in other languages.  KeyId, Algorithm, etc.  You should check another language for the properties on the *Result types.

Reply:
Looking through the Java one, there isn't much returned. Do you have specific examples? We could include the algorithm with encrypt/wrap/sign.

Reply:
Added a lot more fields based on .NET